sinfonier-web/app/views/Secure/layout.html

```diff
@@ -7,6 +7,7 @@
   <link rel="stylesheet" type="text/css" media="screen" href="@{'/public/plugins/bootstrap/css/bootstrap.min.css'}"/>
   <link rel="stylesheet" type="text/css" media="screen" href="@{'/public/plugins/font-awesome/css/font-awesome.min.css'}"/>
   <link rel="stylesheet" type="text/css" media="screen" href="@{'/public/plugins/bootstrap-select/dist/css/bootstrap-select.min.css'}"/>
+  <link rel="stylesheet" type="text/css" media="screen" href="@{'/public/stylesheets/module.css'}"/>
```

Comment:
It's a css imported by darwin, as we replace this template by ours, we currently are missing some styles on the login screen.